explainer.md

```diff
@@ -286,6 +279,39 @@ typically associate with JS.
   * Large ArrayBuffers are not included in usedJSHeapSize and totalJSHeapSize,
     since they are backed by PartitionAlloc.
   * External strings are not included in usedJSHeapSize and totalJSHeapSize
-  
+* The numbers are quantized into buckets and time-delayed.
+
 Moreover performance.memory in Chrome exposes jsHeapSizeLimit which is an JavaScript engine specific
-heuristic which may not exisit in other JavaScript engines. Therefore, we propose to remove this metric.
+heuristic which may not exisit in other JavaScript engines.
+
+# <a name="appendix-d"></a> Appendix D - Frame Tree Examples
+
+Note: All examples assume site isolation - frames from different origins are
+always hosted in different processes.
+
+* Example 1: User opens two tabs to <Origin A>, a simple site with a single
+  frame.
+  * More common case: Browser hosts the two frames in different processes.
+    numberOfFrameTrees is *1* for both sites, and the metrics will be accurate.
+  * Less common case: Browser hosts the two frames in the same process.
+    numberOfFrameTrees is *2* for both sites, and the metrics will be
+    inaccurate.
+    * Dividing by 2 will not always do the right thing! See next example.
```

Comment:
The next example does not really show a problem.

Reply:
Added clear example.